### DISCUSSION TITLE
Help With NordVPN

### DISCUSSION THREAD

**Discussion by tleung927** [haugene/docker-transmission-openvpn#2132] [Q&A]
I was using this docker for years and recently ipvanish  got some problem, it keep crash and sometime it will back to normal. So i decide to give nordvpn a try. 

Here is my install 
docker run --cap-add=NET_ADMIN --device=/dev/net/tun --restart=always -d --name transmission \
-v /mnt/share/:/data \
--env-file DockerEnv \
-e LOCAL_NETWORK=192.168.1.0/24 \
-p 9091:9091 \
haugene/transmission-openvpn

env file
OPENVPN_PROVIDER=NORDVPN
NORDVPN_COUNTRY=US
NORDVPN_CATEGORY=legacy_p2p
NORDVPN_PROTOCOL=tcp
NORDVPN_USERNAME=xxxxxxxxxxxxxxxxxxxx
NORDVPN_PASSWORD=xxxxxxxxxxxxxxxxxxxx
LOCAL_NETWORK=192.168.1.0/24
ENABLE_UFW=true
TRANSMISSION_ALT_SPEED_DOWN=5000 
TRANSMISSION_ALT_SPEED_ENABLED=false 
TRANSMISSION_ALT_SPEED_TIME_BEGIN=540 
TRANSMISSION_ALT_SPEED_TIME_DAY=127 
TRANSMISSION_ALT_SPEED_TIME_ENABLED=false 
TRANSMISSION_ALT_SPEED_TIME_END=1020 
TRANSMISSION_ALT_SPEED_UP=38 
TRANSMISSION_BIND_ADDRESS_IPV4=0.0.0.0 
TRANSMISSION_BIND_ADDRESS_IPV6=:: 
TRANSMISSION_BLOCKLIST_ENABLED=false 
TRANSMISSION_BLOCKLIST_URL=http://www.example.com/blocklist 
TRANSMISSION_CACHE_SIZE_MB=256
TRANSMISSION_DHT_ENABLED=true 
TRANSMISSION_DOWNLOAD_DIR=/data/completed 
TRANSMISSION_DOWNLOAD_LIMIT=100 
TRANSMISSION_DOWNLOAD_LIMIT_ENABLED=0 
TRANSMISSION_DOWNLOAD_QUEUE_ENABLED=true 
TRANSMISSION_DOWNLOAD_QUEUE_SIZE=5 
TRANSMISSION_ENCRYPTION=1 
TRANSMISSION_IDLE_SEEDING_LIMIT=30 
TRANSMISSION_IDLE_SEEDING_LIMIT_ENABLED=false 
TRANSMISSION_INCOMPLETE_DIR=/data/incomplete 
TRANSMISSION_INCOMPLETE_DIR_ENABLED=true 
TRANSMISSION_LPD_ENABLED=false 
TRANSMISSION_MAX_PEERS_GLOBAL=200 
TRANSMISSION_MESSAGE_LEVEL=2 
TRANSMISSION_PEER_CONGESTION_ALGORITHM= 
TRANSMISSION_PEER_ID_TTL_HOURS=6 
TRANSMISSION_PEER_LIMIT_GLOBAL=200 
TRANSMISSION_PEER_LIMIT_PER_TORRENT=100 
TRANSMISSION_PEER_PORT=51413 
TRANSMISSION_PEER_PORT_RANDOM_HIGH=65535 
TRANSMISSION_PEER_PORT_RANDOM_LOW=49152 
TRANSMISSION_PEER_PORT_RANDOM_ON_START=false 
TRANSMISSION_PEER_SOCKET_TOS=default 
TRANSMISSION_PEX_ENABLED=true 
TRANSMISSION_PORT_FORWARDING_ENABLED=false 
TRANSMISSION_PREALLOCATION=1 
TRANSMISSION_PREFETCH_ENABLED=1 
TRANSMISSION_QUEUE_STALLED_ENABLED=true 
TRANSMISSION_QUEUE_STALLED_MINUTES=30 
TRANSMISSION_RATIO_LIMIT=2 
TRANSMISSION_RATIO_LIMIT_ENABLED=false 
TRANSMISSION_RENAME_PARTIAL_FILES=true 
TRANSMISSION_RPC_AUTHENTICATION_REQUIRED=true 
TRANSMISSION_RPC_BIND_ADDRESS=0.0.0.0 
TRANSMISSION_RPC_ENABLED=true 
TRANSMISSION_RPC_PASSWORD=xxxxxx 
TRANSMISSION_RPC_PORT=9091 
TRANSMISSION_RPC_URL=/transmission/ 
TRANSMISSION_RPC_USERNAME=xxxxxxx
TRANSMISSION_RPC_WHITELIST=*.*.*.*
TRANSMISSION_RPC_WHITELIST_ENABLED=true 
TRANSMISSION_SCRAPE_PAUSED_TORRENTS_ENABLED=true 
TRANSMISSION_SCRIPT_TORRENT_DONE_ENABLED=false 
TRANSMISSION_SCRIPT_TORRENT_DONE_FILENAME= 
TRANSMISSION_SEED_QUEUE_ENABLED=false 
TRANSMISSION_SEED_QUEUE_SIZE=10 
TRANSMISSION_SPEED_LIMIT_DOWN=100 
TRANSMISSION_SPEED_LIMIT_DOWN_ENABLED=false 
TRANSMISSION_SPEED_LIMIT_UP=38 
TRANSMISSION_SPEED_LIMIT_UP_ENABLED=true 
TRANSMISSION_START_ADDED_TORRENTS=true 
TRANSMISSION_TRASH_ORIGINAL_TORRENT_FILES=false 
TRANSMISSION_UMASK=2 
TRANSMISSION_UPLOAD_SLOTS_PER_TORRENT=14 
TRANSMISSION_UTP_ENABLED=true 
#TRANSMISSION_WATCH_DIR=/data/watch 
TRANSMISSION_WATCH_DIR_ENABLED=true 
TRANSMISSION_HOME=/data/transmission-home

But i keep getting this error 
stdout: Wed Dec 29 00:13:34 2021 SENT CONTROL [us8585.nordvpn.com]: 'PUSH_REQUEST' (status=1)
stdout: Wed Dec 29 00:13:34 2021 AUTH: Received control message: AUTH_FAILED

I tried to use  email/password and  Service credentials (manual setup) from nordvpn website. But i still getting same  error. Any suggestion?

**Comment by pkishino** [comment]
uhm,
NORDVPN_USERNAME
and NORDVPN_PASSWORD 
are not valid variables..this should be

OPENVPN_USERNAME
OPENVPN_PASSWORD

**Comment by tleung927** [comment]
Oh my bad, that's the testing env, i was put OPENVPN_USERNAME OPENVPN_PASSWORD in my actual file.
The problem now is it will connect sometimes, but pop up same error show auth_failed most of time. Any idea?

**Comment by edgd1er** [comment]
you may try with advanced configuration service credentials:

[![https://nordvpn.nanorep.co/storage/nr1/kb/3E6DB546/3E6DB64A/3E6E3728/12/gdWAxjdVXb.png](https://nordvpn.nanorep.co/storage/nr1/kb/3E6DB546/3E6DB64A/3E6E3728/12/gdWAxjdVXb.png)](https://support.nordvpn.com/Connectivity/Router/1047410472/Tomato-setup-with-NordVPN.htm)

**Comment by tleung927** [comment]
Thanks @edgd1er but i try email and service credential both have same problem. I think it maybe nordvpn server, or the account i bought from ebay is $5.99 for  2 years, i guess this account got ban or something

**Comment by tleung927** [comment]
so weired. It works with tinyproxy, i direct 9091 to 9999 then it works without any problem,
